CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+303 See Other
+
+Location: https://www.elastic.co/community/codeofconduct
```

Comment:
Should this be a link?

Reply:
Markdown will render it as one. If you think my HTTP redirect joke isn't appropriate here, we can of course reword it. But my point was to show that the CoC was located elsewhere and give the user to option to follow the link 😊

Reply:
This is how it will be rendered:

![image](https://user-images.githubusercontent.com/10602/104294639-b8bfdb00-54bf-11eb-8747-f022f2d2914d.png)

Reply:
Ok, that works, but not sure I understand how this is a joke.

Reply:
"303 See Other" is an HTTP redirect for when something is located elsewhere. It's not meant to be read by humans, but most developers will recognize it. But even if you don't, you'll just follow the link anyway, so I thought it wouldn't hurt

Reply:
@gchaps do you think this PR can be merged?